addon-test-support/ember-qunit/legacy-2-x/module-for-component.js

```diff
@@ -3,4 +3,9 @@ import { TestModuleForComponent } from 'ember-test-helpers';
 
 export default function moduleForComponent(name, description, callbacks) {
   createModule(TestModuleForComponent, name, description, callbacks);
+  Ember.deprecate(`The usage "moduleForComponent" is deprecated. Please migrate the "${name}" module to the new test APIs.`, false, {
```

Comment:
* Should probably import `deprecate`
* Can we tell them exactly what to use in each case (e.g. `setupRenderingTest` for this one)?

Reply:
I believe I have resolved this.